tests/WebAssertTest.php

```diff
@@ -255,7 +256,7 @@ public function testResponseHeaderContains()
                 )
             ));
 
-        $this->assertCorrectAssertion('responseHeaderContains', array('foo', 'ba'));
+        $this->assertCorrectAssertion('responseHeaderContains', array('foo', new Stringer('ba')));
```

Comment:
please create a *new* test instead of altering the existing one. We still want to test with an actual string, as that's the documented API.